packages/config/src/android/Orientation.ts

```diff
@@ -16,7 +24,7 @@ export async function setAndroidOrientation(config: ExpoConfig, manifestDocument
   const mainActivity = manifestDocument.manifest.application[0].activity.filter(
     (e: any) => e['$']['android:name'] === '.MainActivity'
   );
-  mainActivity[0]['$'][SCREEN_ORIENTATION_ATTRIBUTE] = orientation;
+  mainActivity[0]['$'][SCREEN_ORIENTATION_ATTRIBUTE] = prepareOrientation(orientation);
```

Comment:
```suggestion
  mainActivity[0]['$'][SCREEN_ORIENTATION_ATTRIBUTE] = orientation !== 'default' ? orientation : 'unspecified';
```
nit pick: prepareOrientation might be a bit misleading and implementation is simple enough to just put inline

Reply:
You're right - assuming that Expo accepts only 3 values: `default`, `portrait` and `landscape`, and 2 of them are valid manifest values, I can simplify that 😉